jobs/demo-district-update-pass.groovy

```diff
@@ -0,0 +1,84 @@
+// Automate monthly change for PL sandbox account passwords
+// https://khanacademy.atlassian.net/browse/DIST-4092
+
+@Library("kautils")
+// Classes we use, under jenkins-jobs/src/.
+import org.khanacademy.Setup;
+// Vars we use, under jenkins-jobs/vars/.  This is just for documentation.
+//import vars.exec
+//import vars.notify
+//import vars.withTimeout
+//import vars.withSecrets
+
+new Setup(steps
+
+).allowConcurrentBuilds(
+
+).addStringParam(
+   "CYPRESS_GIT_REVISION",
+   """The name of a cypress branch to use when building.""",
+   "update-psw"
+
+).addStringParam(
+   "SLACK_CHANNEL",
+   "The slack channel to which to send failure alerts.",
+   "#demo-district-logs"
+
+).addStringParam(
+   "TEST_RETRIES",
+   """How many retry attempts to use. By default is 1.""",
+   "1"
+
+).addCronSchedule(
+   'H H 1 * *'        // Run on the first day of every month
+
+).apply()
+
+// We use the build name as a unique identifier for user notifications. 
+BUILD_NAME = "build demo-district-password-update #${env.BUILD_NUMBER}";
+
+def _setupWebapp() {
+   kaGit.safeSyncToOrigin("git@github.com:Khan/webapp", params.CYPRESS_GIT_REVISION);
+   dir("webapp/services/static") {
+      sh("yarn install");
+   }
+}
+
+def runScript() {
+   def tempDir = "webapp/services/static/javascript/districts-package/__e2e-tests__";
```

Comment:
What makes this a "temp" dir?  This is a directory in our source tree, what's temporary about it?